py3-rpds-py.yaml

```diff
@@ -1,25 +1,38 @@
 package:
   name: py3-rpds-py
   version: 0.20.0
-  epoch: 0
+  epoch: 1
   description: "Python bindings to Rust's persistent data structures (rpds)."
   copyright:
     - license: MIT
   dependencies:
-    runtime:
-      - python3
+    provider-priority: 0
+
+vars:
+  pypi-package: rpds-py
+  module-name: rpds
+
+data:
+  - name: py-versions
+    items:
+      3.10: "310"
+      3.11: "311"
+      3.12: "312"
 
 environment:
   contents:
     packages:
       - build-base
       - busybox
       - ca-certificates-bundle
-      - py3-gpep517
-      - py3-pip
-      - py3-setuptools
-      - py3-wheel
-      - python3
+      - py3-supported-maturin
+      - py3-supported-pip
+      - py3-supported-python
+      - py3-supported-setuptools
+      - py3-supported-wheel
+      # workaround; the build requires the binary but there's no
+      # non-versioned package that would satisfy this requirement.
+      - py3.12-maturin-bin
```

Comment:
https://github.com/wolfi-dev/os/pull/27107 - adds 'maturin' executable rather htan having the 3 'pyXX-maturin-bin'